mozilla_django_oidc/utils.py

```diff
@@ -39,3 +39,19 @@ def is_authenticated(user):
     if _less_than_django_1_10:
         return user.is_authenticated()
     return user.is_authenticated
+
+
+def get_oidc_scopes():
```

Comment:
The main reason for that was to allow people to define the scopes as a list or a string. But if you prefer to just have users configure a string, that works for me.